packages/corvid-cli/test/it/edit.spec.js

```diff
@@ -255,4 +255,49 @@ describe("edit", () => {
       );
     });
   });
+
+  test("should fail on bad .wix file", async () => {
+    const localSiteFiles = localSiteBuilder.buildFull();
+    localSiteFiles["pages"]["page_test.bad_json"] = {
+      "bad_json.wix": "bad_json"
+    };
+    const tempDir = await initTempDir(
+      Object.assign(
+        {
+          ".corvid": { "corvidrc.json": '{ "metasiteId": "12345678" }' }
+        },
+        { src: localSiteFiles }
+      )
+    );
+
+    fetchMock
+      .mock(
+        "https://www.wix.com/_api/corvid-devex-service/v1/listUserSites",
+        JSON.stringify(
+          [
+            {
+              metasiteId: "12345678",
+              publicUrl: "http://a-site.com",
+              siteName: "aSite"
+            }
+          ],
+          null,
+          2
+        )
+      )
+      .mock(
+        `http://frog.wix.com/code?src=39&evid=201&msid=12345678&uuid=testGuid&csi=${
+          process.env.CORVID_SESSION_ID
+        }&status=start`,
+        JSON.stringify({})
+      )
+      .mock(
+        `http://frog.wix.com/code?src=39&evid=201&msid=12345678&uuid=testGuid&csi=${
+          process.env.CORVID_SESSION_ID
+        }&status=success`,
+        JSON.stringify({})
+      );
+
+    return expect(openEditor(tempDir)).rejects.toThrow();
```

Comment:
we need to check that the relevant message is shown to the user

Reply:
done

Reply:
reverted

Reply:
there is no such option now, tried to create, but it was clumsy